geomesa-spark/geomesa-spark-geotools/src/test/scala/org/locationtech/geomesa/spark/geotools/GeoToolsSpatialRDDProviderTest.scala

```diff
@@ -43,7 +44,7 @@ class GeoToolsSpatialRDDProviderTest extends Specification {
       val ds = DataStoreFinder.getDataStore(dsParams)
       ingestChicago(ds)
 
-      val rdd = GeoMesaSpark(dsParams).rdd(new Configuration(), sc, dsParams, new Query("chicago"))
+      val rdd = GeoMesaSpark(dsParams).rdd[SimpleFeature](new Configuration(), sc, dsParams, new Query("chicago"))
```

Comment:
@elahrvivaz we already have a suite of implicits in companion class.  an alternative is to try and play with scope precedence.  e.g. find SimpleFeature first, then find others.